### ISSUE TITLE
[bug]: React Warning: Each child in a list should have a unique "key" prop. in ItemsReview component

### ISSUE THREAD

**Issue by 0m3r** [magento/pwa-studio#3818]
I see react warning in my dev console 

```
react.development.js?72d0:220 Warning: Each child in a list should have a unique "key" prop.

Check the render method of `ItemsReview`. See https://reactjs.org/link/warning-keys for more information.
    at Item (webpack-internal:///./node_modules/@magento/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReview.js:154:14)
    at ItemsReview (webpack-internal:///./node_modules/@magento/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReview.js:286:14)
    at div
```

**To reproduce**

1. Go to checkout
2. Open dev console
3. See warning 

**Expected behavior**

console without simple errors

**Screenshots**

![Знімок екрану з 2022-05-04 14-43-43](https://user-images.githubusercontent.com/412612/166674816-d3cf40cd-a43e-4098-957b-1bd9008967bf.png)

**Possible solutions**

https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReview.js#L37

```diff
diff --git a/packages/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReview.js b/packages/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReview.js
index 02a35694..ea3f1a75 100644
--- a/packages/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReview.js
+++ b/packages/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReview.js
@@ -34,7 +34,7 @@ const ItemsReview = props => {
 
     const items = itemsInCart.map((item, index) => (
         <Item
-            key={item.id}
+            key={item.uid}
             {...item}
             isHidden={!showAllItems && index >= 2}
             configurableThumbnailSource={configurableThumbnailSource}

```

**Please complete the following device information:**

-   Device [e.g. PC, Ubuntu]:
-   Browser [e.g. Chrome]:
-   Browser Version [e.g. 101]:
-   Magento Version [e.g Adobe Commerce 2.4]:
-   PWA-Studio  [e.g Adobe Commerce 12.3-4]:

**Investigation**

After PWA-2309 changing product queries to use uid instead of id

https://github.com/magento/pwa-studio/blame/develop/packages/peregrine/lib/talons/CheckoutPage/ItemsReview/itemsReviewFragments.gql.js

https://github.com/magento/pwa-studio/commit/8df7debeaea86dfb64bc6f5cb327a9d872e3613e

**Please let us know what packages this bug is in regards to:**

-   [ ] `venia-concept`
-   [x] `venia-ui`
-   [ ] `pwa-buildpack`
-   [ ] `peregrine`
-   [ ] `pwa-devdocs`
-   [ ] `upward-js`
-   [ ] `upward-spec`
-   [ ] `create-pwa`

**Comment by anthoula** [comment]
@magento export issue to JIRA project PWA as Bug

**Comment by anthoula** [comment]
@magento export issue to JIRA project PWA as Bug

**Comment by 0m3r** [comment]
Fixed in v12.5.0

https://github.com/0m3r/pwa-studio/blob/v12.5.0/packages/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReview.js

https://github.com/magento/pwa-studio/pull/3856
